ext/featurefmt/apk/testdata/valid.csv

```diff
@@ -0,0 +1,11 @@
+musl,1.1.14-r10,,
+busybox,1.24.2-r9,,
+alpine-baselayout,3.0.3-r0,,
+alpine-keys,1.1-r0,,
+zlib,1.2.8-r2,,
+libcrypto1.0,1.0.2h-r1,openssl,1.0.2h-r1
+libssl1.0,1.0.2h-r1,openssl,1.0.2h-r1
+apk-tools,2.6.7-r0,,
+scanelf,1.1.6-r0,pax-utils,1.1.6-r0
+musl-utils,1.1.14-r10,musl,1.1.14-r10
+libc-utils,0.7-r0,libc-dev,0.7-r0
```

Comment:
why is that? it's easier to read/modify the expected output CSV file if it's directly stored as a file.

Reply:
To avoid complicating the tests with code that loads test files.
I've noticed that in the standard library, they only use `testdata` directories for benchmarks.
I don't feel super strong about it though, since we already do this for a bunch of other tests.

Reply:
Well, in std not very true https://github.com/golang/go/blob/a0d6420d8be2ae7164797051ec74fa2a2df466a1/src/net/interface_linux_test.go#L93
in this case, it's parsing the test file for testing the parser. Basically the same usage as ours. 

Yes, you're right that loading test file will involve extra code complexity.

Reply:
Good find. I think we should play things by ear. If there are many test files or they're in a format that's a pain to edit without tools, we should use a testdata directory with files. If it isn't much of a pain to keep things in the test package in Go source files, we should do that.

I leave the decision up to you.